packages/core/src/extensions/extension-loader/file-system-provisioner-store/get-path-to-legacy-package-json.injectable.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import directoryForUserDataInjectable from "../../../common/app-paths/directory-for-user-data/directory-for-user-data.injectable";
+import joinPathsInjectable from "../../../common/path/join-paths.injectable";
+
+export type GetPathToLegacyPackageJson = (extensionName: string) => string;
+
+const getPathToLegacyPackageJson = getInjectable({
```

Comment:
This could be unit tested as part of `ensure-hashed-directory-for-extension.test.ts`, instead of coupling unit tests too tightly with implementation details.

Reply:
sure we can just get rid off the specific tests for this dependency